examples/module-federation/mf-react-component/tsconfig.json

```diff
@@ -2,7 +2,11 @@
   "compilerOptions": {
     "jsx": "react-jsx",
     "strict": true,
-    "skipLibCheck": true
+    "skipLibCheck": true,
+    "esModuleInterop": true,
+    "paths": {
+      "*": ["./@mf-types/*"]
+    }
   },
-  "include": ["src/**/*"]
+  "include": ["src/**/*", "src/stories"]
```

Comment:
It's better to not put stories inside `src` folder, but beside it.

Reply:
i just follow this repo, do we need to put it beside `src` folder? https://github.com/rspack-contrib/storybook-rsbuild/tree/main/sandboxes/react-16

Reply:
It's better, will reduce the code to ship to NPM. The code in that repository is forked from Storybook's official sandbox repository, while they're not targeting library but app.